### ISSUE TITLE
Deleting an image doesn't delete old formats

### ISSUE THREAD

**Issue by musicpanda** [thirtybees/thirtybees#1541]
When you delete an image it will only delete the formats that are in the database (both active and inactive).  When you had deleted the formats of an old theme images in those formats will not be deleted.

**Comment by Zengraph** [comment]
Yes, you are right, that is why I developped the image cleaning module.. but it's complicated to rlz it for everybody now.
